### PR TITLE
Add log line after jaeger span finish for debugging

### PR DESCRIPTION
**Pull request by madhuravi**

### CODE REVIEW

File: modules/uhttp/filters.go [hunk 1]
```diff
@@ -70,7 +70,10 @@ func tracingServerFilter(host service.Host) FilterFunc {
 		}
 		span := opentracing.GlobalTracer().StartSpan(operationName, ext.RPCServerOption(spanCtx))
 		ext.HTTPUrl.Set(span, r.URL.String())
-		defer span.Finish()
+		defer func() {
+			span.Finish()
+			fxctx.Logger().Info("Span finished")
```
Comment (alsamylkin) [inline modules/uhttp/filters.go]: Add info which span is finished?

File: modules/uhttp/filters_test.go [hunk 2]
```diff
@@ -67,10 +67,13 @@ func TestTracingFilterWithLogs(t *testing.T) {
 		response := testServeHTTP(chain, host)
 		assert.Contains(t, response.Body.String(), "filters ok")
 		assert.True(t, len(buf.Lines()) > 0)
+		var spanFinished bool
 		for _, line := range buf.Lines() {
 			assert.Contains(t, line, "traceID")
 			assert.Contains(t, line, "spanID")
+			spanFinished = strings.Contains(line, "Span finished")
```
Comment (alsamylkin) [inline modules/uhttp/filters_test.go]: We want all spans to be finished, so assert should be moved here?

**Comment by glibsm** [comment]
If it's for debugging, why is it an `info` log and not `debug`?

File: modules/uhttp/filters.go (hunk 1 above)
Comment (madhuravi) [inline modules/uhttp/filters.go]: That information is already added to the logger by the time defer is called. We add trace id and span id to the logger.

File: modules/uhttp/filters_test.go (hunk 2 above)
Comment (madhuravi) [inline modules/uhttp/filters_test.go]: Yes but it's not in all of the log lines emitted ;)

Comment (alsamylkin) [inline modules/uhttp/filters_test.go]: Hmm, so we are checking the last line only? This seems to be an overkill then :)

Comment (madhuravi) [inline modules/uhttp/filters_test.go]: It's minor redundancy in a test Alex :P However the reason is we can't rely on the buf lines order, that would make the test fragile if it's not exactly in the last line and an extra log was added somewhere else.

Comment (alsamylkin) [inline modules/uhttp/filters_test.go]: Well, if it will not be in the last line the test output, the test will fail :)

Comment (madhuravi) [inline modules/uhttp/filters_test.go]: True! Changed to use logical or.

**Pull request merged**